### PR TITLE
feat: standalone conversion and forms page

### PR DESCRIPTION
**Pull request by gokberknur**
This version includes upgrade to angular 16 and standalone architecture conversion (no more ngmodules). 

many angular applications (including scania) are still using NgModules. However, standalone components introduced with angular 14, became fully available with version 15. This is suggested way of developing new apps with angular, so that's why I thought it would be good to do tegel demo page with this way.

In addition to that, form page template introduced, similar to react-demo page. It is not 100% functional yet (and stylings).

### CODE REVIEW

**Review by timrombergjakobsson — APPROVED** [review]
Awesome stuff @gokberknur and thank you so much for your contribution. I am no Angular developer so just focusing on that it compiles. For me it is a go, as different cosmetic stuff and further dev on this we in the team can do. Maybe @theJohnnyMe will have some more constructive feedback on the matter.

File: src/app/components/header/header.component.html [hunk 1]
```diff
@@ -17,6 +17,10 @@
       <a routerLink="/page-two">Page Two</a>
     </tds-header-item>
 
+    <tds-header-item selected="{{ getCurrentUrl() === '/page-two'}}">
```
Comment (theJohnnyMe) [inline src/app/components/header/header.component.html:20]:
```suggestion
    <tds-header-item selected="{{ getCurrentUrl() === '/form'}}">
```
Small change...

File: src/app/directives/dropdown.directive.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (theJohnnyMe) [inline src/app/directives/dropdown.directive.ts:1]: Cool for doing this! I am curious to hear and learn more about directives!

**Comment by theJohnnyMe** [comment]
There is small thing in Header that needs correction, other looks good to me ;)

**Pull request merged**